### ISSUE TITLE
Sample phpcs.xml / hooks should include explicit paths

### ISSUE THREAD

**Issue by leewillis77** [stevegrunwell/wp-enforcer#11]
Context:
- I have phpcs installed already (/usr/local/bin/phpcs).
- I have (an older version) of the WP coding standards installed (Which happens not to have WordPress-Docs)

When setting up with an out-of-the-box install, everything fails since the pre-commit hook is using my "normal" phpcs which knows nothing about the standards under vendor, specifically it knows nothing about WordPress-Docs

I can see the arguments both ways about global installs vs locals, but I think if we're taking the time to download the dependencies then we should set things up to use them. This would allow different projects to use different versions if required.

This would mean prefixing phpcs in the pre-commit hook with an explicit patch, and I think also providing a path for the standards to use, not just the name.

**Comment by stevegrunwell** [comment]
It seems like this conflicts with #3 (which removed the explicit path), so that will require a bit more digging to determine the best approach.

The `installed_paths` directory is being installed relative to the `bin/` directory where PHP_CodeSniffer is installed, so ensuring the local install is used should (theoretically) fix the issue with out-of-date coding standards as a result.

**Comment by leewillis77** [comment]
Hmm, Composer doesn't magically add that to your path. I suspect the originally reporter of #3 meant that the PATH will have vendor/bin prefixed ONLY when running the post-install/update commands - it doesn't get added to your general shell's PATH (AFAIK - it certainly isn't for me)

**Comment by leewillis77** [comment]
So, 

```
"post-install-cmd": [
    "wp-enforcer"
],
"post-update-cmd": [
    "wp-enforcer"
]
```

Is still valid. But anywhere else we should use explicit paths?

**Comment by stevegrunwell** [comment]
@leewillis77 Yeah, `phpcs` is getting called without an explicit prefix [in the actual Git hook that gets copied](https://github.com/stevegrunwell/wp-enforcer/blob/master/bin/hooks/pre-commit#L17). Other than that we should be clean.

**Comment by leewillis77** [comment]
Yes, and I can confirm that changing that to include the specific path to the local phpcs resolves the issue with it being able to find the relevant standards. Happy to raise a PR?

**Comment by stevegrunwell** [comment]
Always happy to look at a PR :)

**Comment by stevegrunwell** [comment]
Fixed by #13, will be released on 0.4.0 :)
